src/hwmon.cpp

```diff
@@ -231,7 +251,7 @@ string HwmonInterface<HwmonT>::lookup()
 			if (paths.size() != 1) {
 				string msg(path + ": ");
 				if (paths.size() == 0) {
-					msg += "Could not find a hwmon with this name: " + name_.value();
+					msg += "Could not find an hwmon with this name: " + name_.value();
```

Comment:
nitpick, `hwmon` stands for `hardware monitor` so perhaps it was correct as `a`. But I'm not `a` English speaker so not really sure :D

Reply:
Depends on how you read it: HWmon sounds like Age-Doubleyou-Mon. The a/an decision is typically done phonologically, so the common thing to use here is "an"